smarkets/tests/streaming_api/client.py

```diff
@@ -103,6 +103,7 @@ def test_login_unauthorized(self):
         except LoginError as ex:
             self.mock_session.disconnect.assert_called_once()
             self.assertEquals(eto.LOGOUT_UNAUTHORISED, ex.reason)
+            self.assertNotEquals(ex.reason_msg, 'Unknown')
```

Comment:
This `assertEquals()` check could pass for infinitely many wrong reasons. Why not turn it into a `assertEquals()`?

Reply:
good point